conf/distro/rpb-wayland.conf

```diff
@@ -11,3 +11,7 @@ DISTRO_FEATURES_remove = "x11"
 PREFERRED_PROVIDER_virtual/egl_hikey = "mali450-userland"
```

Comment:
Like the mesa bbappend, this should probably converted to use a MACHINE_FEATURE. Since this is a .conf and not a recipe, an anonymous python method is needed.